Hunted Mobile/Hunted Mobile/Hunted Mobile/ViewModel/MapViewModel.cs

```diff
@@ -655,6 +674,21 @@ public class MapViewModel : BaseViewModel {
                 await userRepository.Update(mapModel.PlayingUser.Id, mapModel.PlayingUser.Location);
                 Initialized = true;
```

Comment:
```suggestion
                Initialized = true;
                await userRepository.Update(mapModel.PlayingUser.Id, mapModel.PlayingUser.Location);
```
Als het request lang duurt, dan wordt de update ook vaker uitgevoerd. (zie ik nu pas)